.github/workflows/run.yaml

```diff
@@ -83,11 +83,14 @@ jobs:
 
       - name: Run benchmarks
         env:
-            OE_LICENSE: ${{ github.workspace }}/oe_license.txt
+            LICENSE: ${{ secrets.OE_LICENSE }}
         shell: bash -l {0}
         run: |
           set -e
+          echo "$LICENSE" > /tmp/oe_license.txt
```

Comment:
Thanks for the review! Yeah, at least to my understanding, this writes the license to a plain file, but it should be restricted to our instance of the GitHub runner, although it's not encrypted.

The only difference between this and the yammbs CI is where the secret is stored. In yammbs it's `${{ github.workspace }}/oe_license.txt`, which expands to `/home/runner/work/yammbs/yammbs/oe_license.txt`. I chose `/tmp` here to keep it out of the repo in case of any errant `git add` commands. Of course, with the gitignore and removing the license after using it, it should be safer to write it somewhere else if you prefer. `$HOME` could be another option, if we're worried specifically about `/tmp`.

Reply:
Yup all good ideas I think